src/elements/content-sidebar/SidebarSection.js

```diff
@@ -88,6 +88,7 @@ class SidebarSection extends React.PureComponent<Props, State> {
                         data-resin-target={interactionTarget}
                         onClick={this.toggleVisibility}
                         type="button"
+                        aria-expanded={!!isOpen}
```

Comment:
Is the `!!` needed here, since this value should always be defined? Can we keep these attributes sorted alphabetically, as well?

Reply:
@jstoffan I just updated this PR.